service/history/queues/executable.go

```diff
@@ -795,21 +795,30 @@ func EstimateTaskMetricTag(
 	}
 }
 
+var (
+	circuitBreakerExecutableBlockedCounter = metrics.NewCounterDef("circuit_breaker_executable_blocked")
```

Comment:
All of our metric definitions are in `metrics_defs.go` please keep that pattern for now.
I would support per component metrics though if/when we have a use case for that.